docs/arastirma/uret.py

```
@@ -0,0 +1,405 @@
+
+
+yapi = {"bolumler": bolumler, "katmanlar": katmanlar}
+govde = "\n".join(out)
+
+
+# --- sol ray: gezinme AYNI ZAMANDA katman şeması ---
+ray = []
+for b in yapi["bolumler"]:
+    ray.append(f'<a class="rb" href="#{b["id"]}"><span class="rnum">{b["num"]}</span>{b["ad"]}</a>')
+    if b["num"] == "2":
+        ray.append('<div class="yigin" role="group" aria-label="Katman yığını, L0 tabanda">')
```

Ruff (0.16.1)

[warning] String contains ambiguous `ı` (LATIN SMALL LETTER DOTLESS I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)